.vscode/settings.json

```diff
@@ -14,5 +13,6 @@
     "python.linting.mypyEnabled": true,
     "python.linting.mypyArgs": [
         "--config-file=${workspaceFolder}/backend/mypy.ini"
-    ]
+    ],
+    "python.analysis.typeCheckingMode": "basic"
```

Comment:
this is pylance right?

Reply:
It's the built-in type checker, idk what it is 👀